### PR TITLE
Makes https://arethetypeswrong.github.io/ happy

### PR DESCRIPTION
**Pull request by kibertoad**

### CODE REVIEW

File: toad-cache.d.cts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    initForCache(cacheId: string, currentTimeStamp: string): void
+    resetForCache(cacheId: string): void
+}
+
+declare class LruObjectHitStatistics<T> extends LruObject<T>{
+    constructor(max?: number, ttlInMsecs?: number, cacheId?: string, globalStatisticsRecord?: HitStatisticsRecord, statisticTtlInHours?: number);
+}
+
+export {
```
Comment (kibertoad) [inline toad-cache.d.cts:112]: not sure about this syntax, is this the right CommonJS types way?..

Comment (kibertoad) [reply toad-cache.d.cts:112]: see https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FalseESM.md for what this is trying to solve

**Review by gurgunday — APPROVED** [review]
Looks good to me

Will try to test cjs imports locally

**Comment by kibertoad** [comment]
@gurgunday note that they require node16 imports, old school commonjs resolves from "main"/"types"

File: toad-cache.d.cts (hunk 1 above)
Comment (Uzlopak) [reply toad-cache.d.cts:112]: why not doing it like we do it in fastify?

Comment (kibertoad) [reply toad-cache.d.cts:112]: fastify is CJS only: https://arethetypeswrong.github.io/?p=fastify%404.24.3

**Pull request merged**